app/src/main/res/values-lt/strings.xml

```diff
@@ -0,0 +1,275 @@
+<resources>
+    <string name="app_name">FitFlow</string>
+    <string name="home">Namai</string>
+    <string name="activity">Veikla</string>
+    <string name="goals">Tikslai</string>
+    <string name="marketplace">Turgus</string>
+    <string name="waterLogging">Hidratacija</string>
+    <string name="distance">Atstumas</string>
+    <string name="calories">Kalorijos</string>
+    <string name="steps">Žingsniai</string>
+    <string name="user_name">Vardas</string>
+    <string name="profile_creation">Sukurti paskyrą</string>
+    <string name="required_field">*Privalomas laukelis</string>
+    <string name="male">Vyras</string>
+    <string name="female">Moteris</string>
+    <string name="profile_saved">Paskyra išsaugota</string>
+    <string name="gender">Lytis</string>
+    <string name="weight_kg">Svoris (kg)</string>
+    <string name="height_cm">Ūgis (cm)</string>
+    <string name="save">Išsaugoti</string>
+    <string name="user">Vartotojas</string>
+    <string name="settings">Nustatymai</string>
+    <string name="confirm">Patvirtinti</string>
+    <string name="cancel">Atšaukti</string>
+    <string name="enter_your_name_here">Įveskite čia savo vardą</string>
+    <string name="select_your_age">Pasirinkite savo amžių</string>
+    <string name="select">Pasirinkite</string>
+    <string name="select_your_gender">Pasirinkite savo lytį</string>
+    <string name="select_your_weight">Pasirinkite savo svorį</string>
+    <string name="select_your_height">Pasirinkite savo ūgį</string>
+    <string name="something_went_wrong_try_again">Kažkas nutiko. Bandyk iš naujo</string>
+    <string name="Yesterday">vakar</string>
+    <string name="Month">Šį mėnesį</string>
```

Comment:
vakar should be uppercase here
<img src="https://github.com/PVP-K409/fit-flow/assets/127019248/1121a069-8ca0-4789-a8d1-995c8d30e00c" width=50% height=50%>